src/mbed_tools/build/_internal/templates/mbed_config.tmpl

```diff
@@ -7,6 +7,8 @@
 set(MBED_TOOLCHAIN "{{toolchain_name}}" CACHE STRING "")
 set(MBED_TARGET "{{target_name}}" CACHE STRING "")
 set(MBED_CPU_CORE "{{core}}" CACHE STRING "")
+set(MBED_C_LIB "{{c_lib}}" CACHE STRING "")
```

Comment:
Should this be `{{clib}}`? All other c_lib or clib references use `clib` (without the underscore).

Will the existing tests catch this?

Reply:
That should stay like that. It's the others that need to change. Thanks.

Reply:
[Fixed](https://github.com/ARMmbed/mbed-tools/compare/c9ae0ed71dd621ec31cb6e3ed8ef8e3229144c36..295bdcd295136fd731251a66fc4091ec1c1d23ee).